lua/plenary/window/border.lua

```diff
@@ -28,7 +28,7 @@ function Border._create_lines(content_win_options, border_win_options)
   local topright = (right_enabled and border_win_options.topright) or ''
 
   if border_win_options.title then
-    local title = string.format(" %s ", border_win_options.title)
+    local title = border_win_options.title == '' and border_win_options.title or string.format(" %s ", border_win_options.title)
```

Comment:
Can you add some `()` here to make it obvious the order (I always struggle with this).

I'm a little confused about what it's supposed to do at the moment. We could instead check on line 30 maybe something like `if border_win_options.title and border_win_options.title ~= "" then` or similar.

Reply:
That was a ternary operation, but got too long and difficult to read I guess. I will make it more explicit and readable, thanks